visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -371,12 +375,53 @@ public void sauceVisualCheck(String snapshotName, CheckOptions options) {
     }
   }
 
+  private DiffingOptionsIn.Builder setDiffingOptionValue(
+      DiffingOptionsIn.Builder builder, String key, boolean value) {
+    switch (key) {
+      case Content:
+        return builder.withContent(value);
+      case Dimensions:
+        return builder.withDimensions(value);
+      case Position:
+        return builder.withPosition(value);
+      case Structure:
+        return builder.withStructure(value);
+      case Style:
+        return builder.withStyle(value);
+      case Visual:
+        return builder.withVisual(value);
+    }
+    return builder;
+  }
+
+  private DiffingOptionsIn generateDiffingOptions(
+      List<String> enableOnly, List<String> disableOnly) {
+    if (enableOnly != null && !enableOnly.isEmpty()) {
```

Comment:
if `enableOnly` is empty, it should create DiffingOptions with all flags set to `false` (same for `disableOnly`).